src/stateAggregator/accessors/aliasAccessor.ts

```diff
@@ -216,7 +224,7 @@ export const getFileLocation = (): string => join(homedir(), Global.SFDX_STATE_F
 
 const unlockIfLocked = async (fileLocation: string): Promise<void> => {
   try {
-    await unlock(fileLocation);
+    await unlock(fileLocation, { fs });
```

Comment:
the fs is injectable, otherwise it defaults to graceful.

this lets the lockfile use our memfs polyfill